livre/eq/Config.cpp

```diff
@@ -229,11 +229,8 @@ bool Config::frame()
         const uint32_t start = frameMin;
         const uint32_t end = std::max( frameMax, current );
         const int32_t delta = params.animation;
+        const uint32_t interval = end - start;
```

Comment:
are you sure frame number cannot pass the maximum frame id ? Have you tried with the limit ?